pom.xml

```diff
@@ -1080,6 +1089,13 @@
                 <artifactId>swagger-models</artifactId>
                 <version>${swagger.models.version}</version>
             </dependency>
+            <!-- https://mvnrepository.com/artifact/javax.websocket/javax.websocket-api -->
+            <dependency>
+                <groupId>javax.websocket</groupId>
+                <artifactId>javax.websocket-api</artifactId>
+                <version>1.1</version>
```

Comment:
Shall we add the version as a property?

Reply:
Fixed with commit [#342213fd8e3c405a4e6d5dd48c4f7b1f3e6ddce7](https://github.com/wso2/carbon-analytics/pull/1259/commits/342213fd8e3c405a4e6d5dd48c4f7b1f3e6ddce7)